content/knowledge/scalinghistory.md

```diff
@@ -0,0 +1,144 @@
+This was introduced by Stellar.
+In FBA, every node has its “quorum set” which is a list of other nodes which this particular node trusts.
+Therefore, to achieve consensus for a transaction, the node relies on the members in its quorum set.
+As long as the majority of nodes are not malicious, this will ensure security of the system.
+Various overlapping quorum slices across the network makes it almost impossible for the majority of nodes to collude to control consensus.
+Safety and fault tolerance are [preferred over liveness](https://www.youtube.com/watch?v=aU08km2xrz0&ab_channel=Lumenauts).
+In case of an accidental fork, the system is halted until consensus is reached.
+This is important in banking applications.
+A consensus that requires only message exchanging followed by voting process leads to high messages volume per second and less expenditure of electricity than PoW.
+However, a criticism of pure FBA is that it leads to fragile structures of constituent nodes, permitting topology scenarios where a single node failure can cause a network-wide failure.
+For example, [in November 2021](https://u.today/xrp-ledger-is-back-on-track-after-temporary-halt), four validators in Ripple went down which halted the whole network for 15 minutes because consensus could not longer be reached.
+
+## Proof of Stake
+
+Then came the [Proof of Stake](https://www.investopedia.com/terms/p/proof-stake-pos.asp) (PoS) consensus algorithms (e.g. Casper).
+This type no longer needs the computational power used for blocks validating but rather an amount staked by network participants (being single or in clusters).
+Staking is putting some native cryptocurrency as a collateral in order to become a validating entity in the network.
+The higher the stake, the higher the chances of getting to write a block on behalf of the whole ecosystem and in return receive some cryptocurrency.
+Two keywords are different here compared to PoW: (1) Computers that participate in PoS consensus are called Validators, (2) Once a block has been created and accepted by the network, it is said to be forged (not mined).
+Then the validator gets some reward.
+If a user does not want to run his own validator, then he can generally simply "delegate" his tokens to the validators, which is like staking without running his own validator, and add the weight of the delegated tokens to the staked tokens of the chosen validator, in return for giving part of the reward to the validator.
+If a validator tries to be malicious then it gets penalised losing some of its staked cryptocurrency.
+
+One key difference between PoS and PoW is that a block on a PoS chain has deterministic finality, meaning that once it has been accepted fully, there is no way to ever undo it.
+In Pow, it can (in theory) always be undone by creating a longer valid chain of blocks.
```

Comment:
```suggestion
In Pow, it can theoretically always be undone by creating a longer valid chain of blocks.
```

Reply:
Also the 'W' in `PoW` should be capitalized.